scripts/main.js

```diff
@@ -0,0 +1,19 @@
+document.addEventListener('DOMContentLoaded', () => {
+  console.log('imported');
```

Comment:
[OPTIONAL] Kindly make sure you remove console from your code as this may lead to leaking vital information if you continue leaving it. And as you know practice makes a man perfect. Kindly practice removing it.